migrator/migrations/n_56_to_n_57_postgres_groups/legacy/store_impl.go

```diff
@@ -32,7 +32,11 @@ type storeImpl struct {
 // GetAll return all groups currently in the store.
 func (s *storeImpl) GetAll(ctx context.Context) (groups []*storage.Group, err error) {
 	err = s.Walk(ctx, func(g *storage.Group) error {
-		groups = append(groups, g)
+		// There have been occurrences of groups without ID being set within stores. This is partially due to the
+		// failed clean up within the 105_106 migration, so explicitly ignore those groups here as well.
+		if g.GetProps().GetId() != "" {
```

Comment:
The invalid ones are already not usable would it make more sense to just log them and drop them instead of the extra work to migrate them?

Reply:
Yeah added the log statement.

Not sure I get the second part though: they will be dropped and not migrated, essentially `GetAll` from the legacy store will only return the groups that can be migrated (i.e. are not with the old format) and drop all non-usable groups.